src/RefractDataStructure.cc

```diff
@@ -1081,6 +1146,20 @@ namespace drafter {
                 throw snowcrash::Error("unknown type of data structure", snowcrash::MSONError);
         }
 
+        CircularReferenceChecker cr(GetNamedTypesRegistry());
```

Comment:
Grr, there are flaky tests on windows build depended on cucumber. 
I propose to remove compare output of tests in cucumber, all relevant cases are tested in integration tests in C++ code, or remove cucumber tests from  windows build